python/observability-online-eval/pydantic-ai/simple_agent.ipynb

```
@@ -0,0 +1,199 @@
+     "text": [
+      "\u001b[32m[MaximSDK] Initializing Maxim AI(v3.10.8)\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Set up Maxim logger\n",
+    "api_key = os.getenv(\"MAXIM_API_KEY\")\n",
+    "repo_id = os.getenv(\"MAXIM_LOG_REPO_ID\")\n",
+    "\n",
+    "if not api_key:\n",
+    "    print(\"Please set MAXIM_API_KEY environment variable\")\n",
```

Ruff (0.12.2)

Unused function argument: `ctx`

(ARG001)